project_liberation_deployment/README.md

```diff
@@ -0,0 +1,51 @@
+# project-liberation-deployment
+
+Scripts and configuration for project-liberation deployment.
+
+## Cloning the repositories
+
+All below steps/instructions require local copies of `project-liberation` and `project-liberation-config` repositories.
+
+## Deployment
+
+### Prerequisites
+
+#### Ansible
+
+You need `ansible` package for run configuration playbooks. Install it from your system package manager
+
+### Configuring `project-liberation` machine
+
+This step configures machine for project-liberation app.
+
+```bash
+ cd project-liberation/project_liberation_deployment/
+ansible-playbook configure.yml                                    \
+    --inventory ../../project-liberation-config/ansible_inventory \
+    --user      $user
+```
+
+### Deploying to the `project-liberation` machine
```

Comment:
I think there's no need for "to the" in here. "Deploying `project-liberation` machine" should be fine.